src/fdc3/dotnet/DesktopAgent/src/DesktopAgent/Common/PreloadFdc3Names.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Reflection;
+
+namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Common;
+
+public static class PreloadFdc3Names
```

Comment:
'Preload' is something that only makes sense in the context of the embedded web browser. Since these are resource names, you can just call this class `ResourceNames`, and its members accordingly, like `Fdc3Bundle`.